notifier/notifier.go

```diff
@@ -383,25 +391,25 @@ func (n *Notifier) Stop() {
 
 // Describe implements prometheus.Collector.
 func (n *Notifier) Describe(ch chan<- *prometheus.Desc) {
```

Comment:
> I filed an issue on being able to scope a registered with a label before passing it down, i.e. currying, some time ago.

Is that https://github.com/prometheus/client_golang/issues/269 ? I have open questions about that one, and yes, it ties directly into the recommended way of how to do this. Perhaps curried Registerers is the way to go. But let's discuss this elsewhere and in more detail.